Cargo.toml

```diff
@@ -17,7 +17,7 @@ nested-values = ["erased-serde", "serde", "serde_json", "slog/nested-values"]
 [dependencies]
 slog = "2"
 atty = "0.2"
-chrono = { version = "0.4", default-features = false, features = ["clock"] }
+time = "0.2"
```

Comment:
`default-features = false` maybe?

Reply:
I considered this and by a quick glance it seemed not worth it. Or which of the default features are useful to remove in version 2 of time?